src/jit/codegenarm64.cpp

```diff
@@ -4203,6 +4215,7 @@ CodeGen::genJumpTable(GenTree* treeNode)
     getEmitter()->emitIns_R_C(INS_lea,
         emitTypeSize(TYP_I_IMPL),
         treeNode->gtRegNum,
+        REG_NA,
```

Comment:
JumpTable code path is dead (NYI in this function). Will revisit it when it's enabled.